staubli_rx160_support/urdf/rx160l_macro.xacro

```diff
@@ -68,7 +68,7 @@
 			<visual>
 				<origin xyz="0 0 0" rpy="0 0 0"/>
 				<geometry>
-				    <mesh filename="package://staubli_rx160_support/meshes/rx160l/visual/link_4.stl"/>
+					<mesh filename="package://staubli_rx160_support/meshes/rx160l/visual/link_4.stl"/>
```

Comment:
The RosCpp styleguide requires 2 spaces instead of tabs for .cpp and .h files. However; I haven't found anything regarding indentation rules for other files (xacros, launch, urdfs) in the ros ecosystem so I suppose that using 3 spaces instead of 2 is valid.